src/System.ServiceModel.UnixDomainSocket/src/System/ServiceModel/UnixDomainSocketClientCredentialType.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.ServiceModel
+{
+    public enum UnixDomainSocketClientCredentialType
+    {
+        None,
+        Default,
+        Certificate,
+        Windows,
+        IdentityOnly,
+    }
```

Comment:
As we're making a change to add a TransportCredentialOnly security mode, I don't think Only needs to be on here now. Also as it's now clearly only for *nix based systems like mac and Linux, I think PosixIdentity would be clearer, or maybe just Posix as we have just "Windows"